### PR TITLE
F/wpc 522 add font color to peered articles

### PR DESCRIPTION
**Pull request by camilleory**
I added a new field for the font color of peered articles (so that the color can be light when the theme is dark). 
I added the field in the peer info as a colored div as well, so that both theme and font colors get displayed.

### CODE REVIEW

File: packages/api-db-mongodb/src/db/peer.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ export class MongoDBPeerAdapter implements DBPeerAdapter {
       return {
         name: '',
         themeColor: '#000000',
+        themeFontColor: '#d67c15',
```
Comment (penk85) [inline packages/api-db-mongodb/src/db/peer.ts:31]: Do you want this color default?

File: packages/api/src/db/peer.ts [hunk 2]
```diff
@@ -3,7 +3,8 @@ import {RichTextNode} from '../graphql/richText'
 export interface PeerProfile {
   name: string
   logoID?: string
-  themeColor: string
+  themeColor?: string
```
Comment (penk85) [inline packages/api/src/db/peer.ts:6]: Do you want it optional here?

File: packages/editor/src/client/locales/rsuiteEn.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
+// const Calendar = {
```
Comment (penk85) [inline packages/editor/src/client/locales/rsuiteEn.ts:1]: Why did you comment this out here?

Comment (camilleory) [reply packages/editor/src/client/locales/rsuiteEn.ts:1]: This file will be deleted, I don't know why it hasn't been when I updated the branch.

File: packages/api-db-mongodb/src/db/peer.ts (hunk 1 above)
Comment (camilleory) [reply packages/api-db-mongodb/src/db/peer.ts:31]: Maybe not the nicest color indeed, I'll change it to white

File: packages/api/src/db/peer.ts (hunk 2 above)
Comment (camilleory) [reply packages/api/src/db/peer.ts]: I suppose it's more consistant if it's not optional, I changed it

File: packages/editor/src/client/panel/peerEditPanel.tsx [hunk 4]
```diff
@@ -232,6 +232,22 @@ export function PeerEditPanel({id, hostURL, onClose, onSave}: PeerEditPanelProps
...
+                    style={{
+                      backgroundColor: profile?.themeFontColor,
+                      width: '30px',
+                      height: '20px',
+                      padding: '5px',
+                      marginLeft: '5px'
+                    }}></div>
+                </div>
+              </DescriptionListItem>
+              {/* <DescriptionListItem label={t('peerList.panels.themeFontColor')}>
```
Comment (bishoymelek) [inline packages/editor/src/client/panel/peerEditPanel.tsx:248]: Do you still want this?

**Review by bishoymelek — CHANGES_REQUESTED** [review]
🎉 good one, just a simple point.

File: packages/editor/src/client/panel/peerEditPanel.tsx (hunk 4 above)
Comment (camilleory) [reply packages/editor/src/client/panel/peerEditPanel.tsx:248]: No, small forgetfullness on my side

File: examples/website/src/shared/route/articleAdapter.ts [hunk 5]
```diff
@@ -10,6 +10,7 @@ export function peerAdapter(peer: any): Peer {
     logoURL: peer.profile.logo?.squareURL,
     websiteURL: peer.profile.websiteURL,
     themeColor: peer.profile.themeColor,
+    themefontColor: peer.profile.fontColor,
```
Comment (roosnic1) [inline examples/website/src/shared/route/articleAdapter.ts:13]: is this a typo? Or hwy only `fontColor`?

**Review by roosnic1 — COMMENTED** [review]
Looks good justa question

File: examples/website/src/shared/route/articleAdapter.ts (hunk 5 above)
Comment (camilleory) [reply examples/website/src/shared/route/articleAdapter.ts:13]: I thought that would be more precise and prevent confusion if we ever need to use font colors anywhere else in future.

**Review by penk85 — APPROVED** [review]
Looks great

**Review by roosnic1 — APPROVED** [review]
🥳

**Review by bishoymelek — APPROVED** [review]
🎉🎉

**Pull request merged**